### PR TITLE
add storage md

### PR DESCRIPTION
**Pull request by fengshunli**
doc https://github.com/cubefs/cubefs_blog/issues/18

### CODE REVIEW

File: blog/news/storage.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+     "exporterPort": 9503,
+     "prof": "7013"
+}
+```
+
+## The mean of each param is as follows:
+
+| 参数       | 类型         | 含义                           | 必需   |
+|------------|--------------|--------------------------------|--------|
+| role       | string       | 进程角色                       | 是     |
```
Comment (heymingwei) [inline blog/news/storage.md:36]: 中文翻译一下

File: blog/news/storage.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+Before configuring the s3cmd tool, you first need to create a user for the cluster. To execute the command, refer to the user management chapter [2] in the official documentation. After the user is created, you can use its Access Key and Secret Key to configure the s3cmd tool:
+
+```bash
+# s3cmd --configure
+Access Key: 用户AK
+Secret Key: 用户SK
+Default Region [US]: 集群master配置文件的clusterName即集群名称
+S3 Endpoint [s3.amazonaws.com]: 对象网关IP地址+监听端口，如192.168.0.5:17410
+DNS-style bucket+hostname:port template for accessing a bucket [%(bucket)s.s3.amazonaws.com]: 对象网关IP地址/%(bucket)，如192.168.0.5/%(bucket)
+Encryption password: 回车，使用默认值
```
Comment (heymingwei) [inline blog/news/storage.md:67]: 中文翻译一下

File: blog/news/storage.md (hunk 1 above)
Comment (fengshunli) [reply blog/news/storage.md]: done

File: blog/news/storage.md (hunk 2 above)
Comment (fengshunli) [reply blog/news/storage.md]: done

**Comment by heymingwei** [comment]
the file name need to change to "play_with_object_storage" @fengshunli

**Comment by fengshunli** [comment]
> the file name need to change to "play_with_object_storage" @fengshunli

done

**Review by heymingwei — APPROVED** [review]
LGTM

**Comment by heymingwei** [comment]
I find it still need to update a file: blog/README.md，otherwise it can't be display. @fengshunli

**Comment by heymingwei** [comment]
I find it still need to update a file: blog/README.md，otherwise it can't be display. @fengshunli

**Review by heymingwei — APPROVED** [review]

**Comment by NaturalSelect** [comment]
@fengshunli Please resolve conflicts.

**Comment by fengshunli** [comment]
@NaturalSelect  done

**Review by NaturalSelect — APPROVED** [review]
LGTM

**Pull request merged**